src/config/backpack/basset.php

```diff
@@ -5,11 +5,11 @@
     'dev_mode' => env('BASSET_DEV_MODE', env('APP_ENV') === 'local'),
 
     // disk and path where to store bassets
-    'disk' => 'public',
+    'disk' => env('BASSET_DISK', 'public'),
```

Comment:
Let's document these new .ENV variables please. And ideally also create a section in the README called "How to use on Laravel Vapor"